resources/crds/hammerdb_crds/postgres/ripsaw_v1alpha1_hammerdb_postgres_cr.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: hammerdb-benchmark
+  namespace: my-ripsaw
+spec:
+  elasticsearch:
+    url: http://my.elasticsearch.server:80
+    index_name: ripsaw-hammerdb
+  metadata:
+    collection: false
+  workload:
+    name: hammerdb
+    args:
+      #image: quay.io/ebattat/hammerdb:latest
```

Comment:
We should clean out these left-overs from development

Reply:
Hi,

I have fixed it for all the crds:
Left empty for future use
#image: # add custom hammerdb image